__tests__/integration.test.js

```diff
@@ -16,8 +16,8 @@ describe('selectTopics()', () => {
       .then(({ body }) => {
         expect(body.topics).toHaveLength(3);
         expect(body.topics[0]).toMatchObject({
-          slug: 'mitch',
-          description: 'The man, the Mitch, the legend',
+          slug: expect.any(String),
```

Comment:
best to do this for all objects being tested! so that we know all objects have the right shape